controllers/defaults/placements.go

```diff
@@ -69,7 +72,8 @@ var (
 				getOcsToleration(),
 			},
 			TopologySpreadConstraints: []corev1.TopologySpreadConstraint{
-				getTopologySpreadConstraintsSpec(1),
+				getTopologySpreadConstraintsSpec(1, osdPrepareLabelSelector),
+				getTopologySpreadConstraintsSpec(1, osdLabelSelector),
```

Comment:
Instead of multiple TSCs, I think we need to have a single TSC with multiple `MatchExpressions`, where one expression is for `osdPrepareLabelSelector ` and one for `osdLabelSelector`.